tensorflow/lite/python/lite.py

```diff
@@ -344,7 +344,7 @@ def from_concrete_functions(cls, funcs):
         message = "This function takes in a list of ConcreteFunction."
         if isinstance(func, _def_function.Function):
           message += (" To get the ConcreteFunction from a Function,"
-                      " call from_concrete_function.")
+                      " call get_concrete_function.")
```

Comment:
can you please check below error reported internally.
` File "<embedded stdlib>/unittest/case.py", line 59, in testPartExecutor
   yield
 File "<embedded stdlib>/unittest/case.py", line 605, in run
   testMethod()
 File "/google3/runfiles/google3/third_party/tensorflow/python/framework/test_util.py", line 1343, in decorated
   return f(self, *args, **kwargs)
 File "/google3/runfiles/google3/third_party/tensorflow/lite/python/lite_v2_test.py", line 119, in testTypeInvalid
   self.assertIn('call from_concrete_function', str(error.exception))
 File "<embedded stdlib>/unittest/case.py", line 1089, in assertIn
   self.fail(self._formatMessage(msg, standardMsg))
 File "/google3/runfiles/google3/third_party/py/absl/testing/absltest.py", line 1767, in fail
   return super(TestCase, self).fail(self._formatMessage(prefix, msg))
 File "<embedded stdlib>/unittest/case.py", line 670, in fail
   raise self.failureException(msg)
AssertionError: 'call from_concrete_function' not found in 'This function takes in a list of ConcreteFunction. To get the ConcreteFunction from a Function, call get_concrete_function.
`

Reply:
@rthadur I fixed the failing internal unittest in 359f22eef2e6c6e0adc804f88b1a94b91f7e2372

It was directly asserting on the content of the error message.